mobile/src/androidTest/java/ch/epfl/sdp/mobile/test/state/StatefulHomeTest.kt

```diff
@@ -66,4 +66,27 @@ class StatefulHomeTest {
     rule.onNodeWithText(strings.sectionSocial).assertIsSelected()
     rule.onNodeWithText(strings.sectionSettings).assertIsNotSelected()
   }
+
+  @Test
+  fun clickPlaySection_selectsPlaySection() = runTest {
+    val facade = AuthenticationFacade(emptyAuth(), emptyStore())
+    facade.signUpWithEmail("email", "name", "password")
+    val user = facade.currentUser.filterIsInstance<AuthenticatedUser>().first()
+    val strings = rule.setContentWithLocalizedStrings { StatefulHome(user) }
+    rule.onNodeWithText(strings.sectionPlay).performClick()
+    rule.onNodeWithText(strings.sectionPlay).assertIsSelected()
+    rule.onNodeWithText(strings.sectionSocial).assertIsNotSelected()
+  }
+
+  @Test
+  fun clickNewGame_inPlaySection_switchesToBoard() = runTest {
+    val facade = AuthenticationFacade(emptyAuth(), emptyStore())
+    facade.signUpWithEmail("email", "name", "password")
+    val user = facade.currentUser.filterIsInstance<AuthenticatedUser>().first()
+    val strings = rule.setContentWithLocalizedStrings { StatefulHome(user) }
+    rule.onNodeWithText(strings.sectionPlay).performClick()
+    rule.onNodeWithText(strings.sectionPlay).assertIsSelected()
+    rule.onNodeWithText(strings.sectionSocial).assertIsNotSelected()
+    rule.onNodeWithText(strings.newGame).assertExists().performClick().assertDoesNotExist()
```

Comment:
As you can see on Cirrus, some test fail. I search a bit in your branch and the report says that the Social API is missing.

Reply:
Thanks it's fixed now